new/LIT7061Qene.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7061Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድንግለ፡ እስራኤል፡ ናንክር፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Dǝngǝla ʾƎsrāʾel nānkǝr (Qǝne of the Mawaddǝs-type)</title>
+                <title xml:lang="en" corresp="#t1">The Virgin of Israel, we admire …</title>
```

Comment:
"we should admire" (jussive). Or, to make a bit more sense out of it "We should admire the Virgin of Israel ..."